src/main/java/algorithm/VisualizationManager.java

```diff
@@ -0,0 +1,395 @@
+package algorithm;
+
+import com.interactivemesh.jfx.importer.stl.StlMeshImporter;
+import com.jme3.math.Quaternion;
+import inputOutput.Tool;
+import javafx.scene.*;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.paint.Color;
+import javafx.scene.paint.PhongMaterial;
+import javafx.scene.shape.Mesh;
+import javafx.scene.shape.MeshView;
+import javafx.scene.transform.Rotate;
+import javafx.scene.transform.Translate;
+import javafx.stage.FileChooser;
+import javafx.stage.Stage;
+import shapes.ConeMesh;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * This class is used to manage the 3D-Visualization.
+ */
+public class VisualizationManager {
+
+    Label statusLabel;
```

Comment:
Unless you specifically need package-private as access modifier, we could make this private like the rest (also in other places I did not mark explicitly)
```suggestion
    private Label statusLabel;
```